Documentation/api/dap/README.md

```diff
@@ -0,0 +1,107 @@
+# DAP Interface
+
+Delve exposes a [DAP](https://microsoft.github.io/debug-adapter-protocol/overview) API interface.
+
+This interface is served over a streaming TCP socket, *not* over HTTP when the `dlv` server is run in one of the two headless modes:
```

Comment:
When is it ever served over HTTP? Is there even a DAP over HTTP?

Reply:
The JSON-RPC documentation says that because JSON-RPC (version 1) can be served either over TCP or over HTTP.